.bin/esbuild.mjs

```diff
@@ -17,7 +17,9 @@ const files = [
 	},
 	{ in: 'assets/js/admin.js', out: 'admin.min' },
 	{ in: 'assets/js/default-calendar.js', out: 'default-calendar.min' },
+	{ in: 'assets/js/sc-oauth-helper-admin.js', out: 'sc-oauth-helper-admin.min' },
```

Comment:
We should be consistent with naming, why `sc-` prefix in name when we don't have in other files?

Reply:
Remove SC-